samples/Tests/Action.NoBody.json

```diff
@@ -0,0 +1,40 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
```

Comment:
>	 [](start = 0, length = 1)

do we care about `\t` in json files?

Reply:
Half of them are \t the other half are spaces. So not currently.